onnx_caffe2/backend.py

```diff
@@ -147,21 +105,10 @@ class Caffe2Backend(Backend):
         'GlobalAveragePool':    'AveragePool',
     }
 
-    _conv_transpose_unpool_renamed_attrs  = {'kernel_shape': 'kernels'}
-    _conv_pool_op_renamed_attrs           = {'kernel_shape': 'kernels'}
-
-    # NB: domain is ONNX operator names.  This only really makes
-    # sense in the context of _renamed_operators as well
-    _renamed_attrs = {
+    _global_renamed_attrs = {'kernel_shape': 'kernels'}
```

Comment:
I'm not sure about the wisdom of "deduplicating" the attribute conversion here. Yes, it looks duplicated, but that's because the actual Caffe2 code we are mapping to is literally two distinct codepaths `conv_pool_op_base.h` and `conv_transpose_unpool_op_base.h`; so logically, it makes sense that we should duplicate it here. And so unifying them will suggest to people that a change in attribute naming for `ConvTranspose` will apply to `Transpose` at all, which is not a priori true.

Reply:
This conversion true globally, caffe2 uses "kernels" consistently in all ops and onnx will use "kernel_shape" consistently in all ops.

Reply:
Well, I'm willing to let this one lie... but "global" spec constraints need to be communicated/remembered  by everyone so we don't violate them ;)